prow/plugins/config.go

```diff
@@ -1161,6 +1166,17 @@ func compileRegexpsAndDurations(pc *Configuration) error {
 	}
 	pc.CherryPickUnapproved.BranchRe = branchRe
 
+	for _, blockade := range pc.Blockades {
+		if blockade.BranchRegexp == nil {
+			continue
+		}
+		branchRe, err := regexp.Compile(*blockade.BranchRegexp)
```

Comment:
maybe we could add to `config_test` a failing regex to ensure we catch it?

Reply:
otherwise LGTM

Reply:
> maybe we could add to config_test a failing regex to ensure we catch it?

@matthyx do you mind if we move this to a follow-up? We currently don't have any tests in `config_test.go` that check for failing regexes so adding a test for `compileRegexpsAndDurations` would also require validating other plugin configs with regex (not just blockade).